moesifwsgi/send_async_event.py

```diff
@@ -0,0 +1,44 @@
+
+class SendEventAsync:
+    def exit_handler(self, scheduler, debug):
+        try:
+            # Shut down the scheduler
+            scheduler.shutdown()
+        except:
+            if debug:
+                print("Error while closing the queue or scheduler shut down")
+
+    def send_event(self, api_client, batch_events, debug):
```

Comment:
should it be send_events instead of send_event?

Reply:
Renamed the function name